MODULE.bazel

```diff
@@ -0,0 +1,26 @@
+module(
+    name = "swiftlint",
+    version = "0.50.3",
+    compatibility_level = 1,
+    repo_name = "SwiftLint",
+)
+
+bazel_dep(name = "platforms", version = "0.0.5")
+bazel_dep(name = "rules_apple", version = "2.0.0", repo_name = "build_bazel_rules_apple")
+bazel_dep(name = "rules_swift", version = "1.5.1", repo_name = "build_bazel_rules_swift")
+bazel_dep(name = "rules_xcodeproj", version = "0.12.2", repo_name = "com_github_buildbuddy_io_rules_xcodeproj")
```

Comment:
Do we have any benefits from renaming these repos? Can we just use the default names?

Reply:
for folks still using the WORKSPACE on any names that would overlap i think we need to keep the old names, on the ones that are much more specifically named we could change it as long as we sync'd that with the workspace downloads, this was the path of least resistance